### PR TITLE
Removed PnH code

### PR DESCRIPTION
**Pull request by psiinon**
Fixes  zaproxy/zaproxy#5136

### CODE REVIEW

**Comment by kingthorin** [comment]
https://github.com/zaproxy/zap-extensions/blob/f18ad5fdcd27aa9bf0bf632086127cfe8ca7082a/src/org/zaproxy/zap/extension/quickstart/resources/help/contents/quickstart.html#L34-L36 Should be removed too.

**Comment by psiinon** [comment]
Good spot!

**Comment by kingthorin** [comment]
Also (there might be others that are less obvious):
https://github.com/zaproxy/zap-extensions/blob/f18ad5fdcd27aa9bf0bf632086127cfe8ca7082a/src/org/zaproxy/zap/extension/quickstart/resources/Messages.properties#L10-L11

https://github.com/zaproxy/zap-extensions/blob/f18ad5fdcd27aa9bf0bf632086127cfe8ca7082a/src/org/zaproxy/zap/extension/quickstart/resources/Messages.properties#L38

https://github.com/zaproxy/zap-extensions/blob/f18ad5fdcd27aa9bf0bf632086127cfe8ca7082a/src/org/zaproxy/zap/extension/quickstart/resources/Messages.properties#L81-L82

**Comment by psiinon** [comment]
Even better :)

File: src/org/zaproxy/zap/extension/quickstart/pnh/ExtensionQuickStartPnH.java [hunk 1]
```diff
@@ -1,264 +0,0 @@
...
-        if (confField == null) {
-            return;
-        }
-        // PnH URL Field has the same enable state as the API
-        confField.setEnabled(apiState);
-        if (apiState) {
-            confField.setText(getPlugNHackUrl());
-        } else {
-            confField.setText(Constant.messages
-                    .getString("quickstart.mitm.api.disabled"));
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/quickstart/pnh/ExtensionQuickStartPnH.java]: This one was not removed.

Comment (psiinon) [reply src/org/zaproxy/zap/extension/quickstart/pnh/ExtensionQuickStartPnH.java]: Now removed :)

**Comment by thc202** [comment]
Thanks!

**Pull request merged**